src/components/text/_text.scss

```diff
@@ -1,175 +1,45 @@
-
 $include-html: false !default;
 
 @if ($include-html) {
 
-  .mint-text-bit {
-    color: $textBitColor;
-    font-family: $textBitFontFamily;
-    font-size: $textBitFontSize;
-    line-height: $textBitLineHeight;
-    text-transform: uppercase;
-    word-wrap: break-word;
-    margin: 0;
-    position: relative;
-    letter-spacing: -1px;
-    &__hole {
-      display: block;
-      position: absolute;
-      top: -22px;
-      left: -13px;
-      line-height: 0;
-      &--small {
-        top: -25px;
-        left: -10px;
-      }
-    }
-    &--alt {
-      color: $textBitAltColor;
-    }
-    &--light {
-      color: $textBitLightColor;
-    }
-    &--dark {
-      color: $textBitDarkColor;
-    }
-    &--warning {
-      color: $textBitWarningColor;
-    }
-    &--large {
-      font-size: $textBitLargeFontSize;
-    }
-    @media (min-width: $breakpointTablet) {
-      &:not(.mint-text-bit--not-responsive) {
-        font-size: $textBitLargeFontSize;
-      }
-    }
-    &--to-larger:not(.mint-text-bit--not-responsive) {
-      @media (min-width: $breakpointTablet) {
-        font-size: $textBitLargerTabletFontSize;
-      }
-      @media (min-width: $breakpointDesktop) {
-        font-size: $textBitLargerDesktopFontSize;
-      }
-    }
-  }
-
-  .mint-text-block {
-    font-family: $textBlockFontFamily;
-    font-size: $textBlockFontSize;
-    line-height: $textBlockLineHeight;
-    color: $textBlockColor;
-    margin: 0;
-    padding: 0;
+  .mint-text {
+    @include typeVariant(default, 1);
+    font-family: $fontFamilyPrimary;
+    color: $black;
 
-    &--gray {
-      color: $textBlockGrayColor;
-    }
-    &--light {
-      color: $textBlockLightColor;
-    }
-    &--for-fine-print-light {
-      color: $textBlockLightBlueColor;
-    }
-    &--for-fine-print {
-      color: $textBlockBlueColor;
-    }
-    &--emphasised {
-      text-transform: uppercase;
+    &--standout {
+      @include typeVariant(standout, 1);
     }
-  }
-
-  .mint-text-description {
-    font-family: $textDescriptionFontFamily;
-    font-size: $textDescriptionFontSize;
-    line-height: $textDescriptionLineHeight;
-    color: $textDescriptionColor;
-    margin: 0;
-    padding: 0;
 
-    &--large {
-      font-size: $textDescriptionLargeFontSize;
-      line-height: $textDescriptionLargeLineHeight;
-    }
-    &--small {
-      font-size: $textDescriptionSmallFontSize;
-      line-height: $textDescriptionSmallLineHeight;
+    &--obscure {
+      @include typeVariant(obscure, 1);
     }
-  }
-
-  .mint-text-emphasised {
-    font-family: $textEmphasisedFontFamily;
-  }
 
-  .mint-link {
-    cursor: pointer;
-    color: $linkColor;
-    text-decoration: none;
-    font-family: $linkFontFamily;
+    &--headline {
+      @include typeVariant(headline, 1.334);
+      font-weight: normal;
+      margin: 0;
+      margin-bottom: rhythm(1);
+    }
 
-    &:hover, &:active {
-      text-decoration: underline;
+    &--emphasised {
```

Comment:
Maybe its a good idea to add `<strong> <em> <b>` styles inside `mint-text`, as those things are used only for text styling...